cpp/examples/abm_parameter_study.cpp

```diff
@@ -0,0 +1,228 @@
+    auto validity_period       = mio::abm::days(1);
+    auto probability           = 0.5;
+    auto start_date            = mio::abm::TimePoint(0);
+    auto end_date              = mio::abm::TimePoint(0) + mio::abm::days(10);
+    auto test_type             = mio::abm::TestType::Antigen;
+    auto test_parameters       = model.parameters.get<mio::abm::TestData>()[test_type];
+    auto testing_criteria_work = mio::abm::TestingCriteria();
+    auto testing_scheme_work   = mio::abm::TestingScheme(testing_criteria_work, validity_period, start_date, end_date,
+                                                         test_parameters, probability);
+    model.get_testing_strategy().add_scheme(mio::abm::LocationType::Work, testing_scheme_work);
+
+    // Assign infection state to each person.
+    // The infection states are chosen randomly with the following distribution
+    std::vector<ScalarType> infection_distribution{0.5, 0.3, 0.05, 0.05, 0.05, 0.05, 0.0, 0.0};
+    for (auto& person : model.get_persons()) {
+        mio::abm::InfectionState infection_state = mio::abm::InfectionState(
+            mio::DiscreteDistribution<size_t>::get_instance()(mio::thread_local_rng(), infection_distribution));
+        auto person_rng = mio::abm::PersonalRandomNumberGenerator(person);
+        if (infection_state != mio::abm::InfectionState::Susceptible) {
+            person.add_new_infection(mio::abm::Infection(person_rng, mio::abm::VirusVariant::Wildtype, person.get_age(),
+                                                         model.parameters, start_date, infection_state));
+        }
+    }
+
+    // Assign locations to the people
+    for (auto& person : model.get_persons()) {
+        const auto id = person.get_id();
+        //assign shop and event
+        model.assign_location(id, event);
+        model.assign_location(id, shop);
+        //assign hospital and ICU
+        model.assign_location(id, hospital);
+        model.assign_location(id, icu);
+        //assign work/school to people depending on their age
+        if (person.get_age() == age_group_5_to_14) {
+            model.assign_location(id, school);
+        }
+        if (person.get_age() == age_group_15_to_34 || person.get_age() == age_group_35_to_59) {
+            model.assign_location(id, work);
+        }
+    }
+
+    // During the lockdown, social events are closed for 90% of people.
+    auto t_lockdown = mio::abm::TimePoint(0) + mio::abm::days(10);
+    mio::abm::close_social_events(t_lockdown, 0.9, model.parameters);
+
+    return model;
+}
+
+int main()
+{
+    mio::mpi::init();
+
+    mio::set_log_level(mio::LogLevel::warn);
+
+    // Set start and end time for the simulation.
+    auto t0   = mio::abm::TimePoint(0);
+    auto tmax = t0 + mio::abm::days(5);
+    // auto sim  = mio::abm::Simulation(t0, std::move(model));
+    const size_t num_runs = 3;
+
+    // Create a parameter study. The ABM currently does not use parameters or dt, so we set them both to 0.
```

Comment:
```suggestion
    // Create a parameter study. The ABM currently does not use parameters or dt, so we set them both to 0.
```
What do you mean with "the ABM does not use parameters"?